diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/IndentationRule.kt

```diff
@@ -188,11 +191,33 @@ class IndentationRule(private val configRules: List<RulesConfig>) : Rule("indent
         if (checkResult?.isCorrect != true && expectedIndent != indentError.actual) {
             WRONG_INDENTATION.warnAndFix(configRules, emitWarn, isFixMode, "expected $expectedIndent but was ${indentError.actual}",
                 whiteSpace.startOffset + whiteSpace.text.lastIndexOf('\n') + 1, whiteSpace.node) {
+                // if it is triple-quoted string template we need to indent all its parts
+                if (whiteSpace.node.treeNext.elementType == STRING_TEMPLATE &&
+                        whiteSpace.node.treeNext.text.startsWith("\"\"\"")) {
+                    fixStringLiteral(whiteSpace, expectedIndent)
+                }
                 whiteSpace.node.indentBy(expectedIndent)
             }
         }
     }
 
+    @Suppress("MagicNumber")
+    private fun fixStringLiteral(whiteSpace: PsiWhiteSpace, expectedIndent: Int) {
+        val textIndent = " ".repeat(expectedIndent + 4)
```

Comment:
Instead of `4` you should use a constant from configuration

Reply:
Fixed